### DISCUSSION TITLE
Циклическое перенаправление на странице

### DISCUSSION THREAD

**Discussion by psierkov** [leon-mbs/zstore#397] [Q&A]
После установки не получается зайти,
браузер выдает ошибку - Циклическое перенаправление на странице

/ перенаправляет на /index.php?q=p:App/Pages/UserLogin
/index.php?q=p:App/Pages/UserLogin перенаправляет на /

и так по кругу, никакого содержимого в браузере не отображается

**Comment by leon-mbs** [comment]
такая   фигня  иногда  случается при первлом открытии   не могу  воспроизвести. эту  ошибку  что то  связано с  браузером  то ли  куки не  пишутся  толи не  читаются.
Попробуйте  почистить кеш  браузера  или  попробуйте  другим  браузером
можно  еще  в папке  logs  глянуть  вдруг там кааая  ошибка

**Reply by psierkov** [reply]
Ошибка была связана с тем что сервер не мог сохранять у себя временные файлы для хранения coockie.

**Reply by leon-mbs** [reply]
ну  то не  куки  а  сессии  но  все  равно странно.  он же  должен  папку  для  временных файлов  пользовать
